distribution/docker/README.md

```diff
@@ -0,0 +1,123 @@
+[![Build Status](https://dev.azure.com/knotx/Knotx/_apis/build/status/Knotx.knotx-starter-kit?branchName=master)](https://dev.azure.com/knotx/Knotx/_build/latest?definitionId=3&branchName=master)
```

Comment:
Please clean this file. It should contain details about:
- how to build
- how to run
- what is the expected result
- link to the tutorial
- references

Reply:
✔️ done